rollup.config.js

```diff
@@ -1,21 +1,33 @@
-import commonjs from 'rollup-plugin-commonjs';
-import nodeBuiltins from 'rollup-plugin-node-builtins';
-import nodeResolve from 'rollup-plugin-node-resolve';
-import babel from 'rollup-plugin-babel';
+'use strict';
 
-export default {
+const commonjs = require('rollup-plugin-commonjs');
+const nodeGlobals = require('rollup-plugin-node-globals');
+const nodeBuiltins = require('rollup-plugin-node-builtins');
+const nodeResolve = require('rollup-plugin-node-resolve');
+const jsonPlugin = require('rollup-plugin-json');
+const babel = require('rollup-plugin-babel');
+
+const onwarn = warning => {
```

Comment:
What is this doing?

Reply:
The tests generated a lot of noisy & repetitive warnings that we wanted to silence - specifically, warnings for circular dependencies and invocations of `eval`. 

When we ran the tests, we were getting an entire pageful of repetitive Circular Dependency warnings originating from within `node_modules` that looked like:

`Circular dependency: node_modules/chai/lib/chai.js -> node_modules/chai/lib/chai/utils/index.js -> node_modules/chai/lib/chai/utils/addMethod.js -> node_modules/chai/lib/chai.js`

These circular dependencies were within modules like chai, so we deemed them non-concerns. Similarly, the following warning about using `eval` was printed over 20 times:

`lib/bson/parser/deserializer.js (506:4) Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification`

We decided to silence this one as well. Are the concerns about using `eval` valid for in-browser use? Should we be concerned?

Reply:
Ok, that makes sense. Can you move this to the `karma.conf.js`, so it is only added for the tests, and not for the build step?